crates/re_log_types/src/data_cell.rs

```diff
@@ -380,7 +380,9 @@ impl DataCell {
     /// Fails if the underlying arrow data cannot be deserialized into `C`.
     #[inline]
     pub fn try_to_native<'a, C: Component + 'a>(&'a self) -> DataCellResult<Vec<C>> {
-        re_tracing::profile_function!(C::name().as_str());
+        // TODO(#3850): Don't, it's way too much and will therefore lie to you.
```

Comment:
I suggest changing this comment to something more explicit like:

`// TODO(#3850): Don't add a profile scope here: the profiler overhead is too big for this fast function"